docs/index.md

```diff
@@ -90,7 +90,7 @@ Terraform 0.13 and later:
 terraform {
   required_providers {
     juju = {
-      version = "~> 0.11.0"
+      version = "~> 0.12.0"
```

Comment:
this is a generated file, so the actual changes are somewhere else.

Reply:
yeah I didn't edit this file by hand, go generate made these changes based on my changes on the examples and the index template.